src/features/island/collectibles/components/HeartBalloons.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import heartBalloons from "assets/events/valentine/sfts/balloons.webp";
+import { PIXEL_SCALE } from "features/game/lib/constants";
+
+export const HeartBalloons: React.FC = () => {
+  return (
+    <>
+      <img
+        src={heartBalloons}
+        style={{
+          width: `${PIXEL_SCALE * 34}px`,
```

Comment:
<img width="305" alt="Screen Shot 2023-02-08 at 4 56 54 pm" src="https://user-images.githubusercontent.com/11745561/217446871-659fca37-c690-4ec2-90a9-283dc064f6cc.png">

`18` is the desired width 😄